### PR TITLE
documentaion update

### PR DESCRIPTION
**Pull request by andreasonny83**
user-interactions.md updated

### CODE REVIEW

File: app/authoring/user-interactions.md [hunk 1]
```diff
@@ -23,13 +23,19 @@ The `prompt` method is asynchronous and return a promise. You'll need to return
...
       name    : 'name',
       message : 'Your project name',
       default : this.appname // Default to current folder name
-    }).then(function (answers) {
-      this.log(answers.name);
+    }, {
+      type    : 'confirm',
+      name    : 'cool',
+      message : 'Would you like to enable the Cool feature?',
+      default : this.cool
```
Comment (SBoudrias) [inline app/authoring/user-interactions.md]: Let's remove that line as copy/pasting wouldn't just work.

Comment (andreasonny83) [inline app/authoring/user-interactions.md]: As discussed with @SBoudrias, the documentation is inconsistent as it shows only a special scenario where the `prompt` array contains only one item. I extended the documentation to show a more real case in where the variable contains two questions trying to help other developers reading for the first time this tutorial

Comment (SBoudrias) [inline app/authoring/user-interactions.md]: I'm all for two question, let's just not add a `default` to the second one as `this.cool` doesn't exists by default.

**Comment by andreasonny83** [comment]
right, my mistake.
It should be ok now

**Comment by SBoudrias** [comment]
Awesome, thanks!

**Pull request merged**